src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -102,6 +102,11 @@ public interface IPreferences
         /// </summary>
         List<string> RecentFiles { get; set; }
 
+        /// <summary>
+        /// Returns path of the Backup location
+        /// </summary>
+        string BackupLocation { get; set; }
```

Comment:
Can you remind me if we can add things to the interface in core? I think this is API breaking right? @mjkkirschner @pinzart

Reply:
Keeping the property in the class.